pennylane/_qubit_device.py

```diff
@@ -84,6 +84,8 @@ class QubitDevice(Device):
     _tensordot = staticmethod(np.tensordot)
     _conj = staticmethod(np.conj)
     _imag = staticmethod(np.imag)
+    _roll = staticmethod(np.roll)
+    _stack = staticmethod(np.stack)
```

Comment:
Just double checking, but something I noticed in previous PRs is that the three frameworks don't have consistency in how they implement 'stacking'. e.g., if I recall, `torch.stack()` and `tf.stack()` were roughly equivalent to `np.vstack()`, and didn't have anything analogous to `stack`?

Reply:
Nice!